CoreUtilities/src/au/gov/asd/tac/constellation/utilities/text/SpellCheckingTextArea.java

```diff
@@ -227,4 +230,161 @@ public void autoComplete(final List<String> suggestions) {
             }
         });
     }
+    
```

Comment:
These functions can be removed if the key pressed event handling are redundant and can be removed

Reply:
Removed and verified